docs/quickstarts/deploy-dicom-cast.md

```diff
@@ -2,16 +2,22 @@
 
 In this quickstart, you will learn how to deploy DICOM Cast for your Medical Imaging Server for DICOM. DICOM Cast is a service which pushes Medical Imaging Server for DICOM metadata into a FHIR&trade; server to support integrated queries across clinical and imaging data. To learn more about the architecture of DICOM Cast, refer to the [DICOM Cast Concept](../concepts/dicom-cast.md).
 
-DICOM Cast is deployed as an Azure Container Instance using the provided [ARM template](/converter/dicom-cast/samples/templates/default-azuredeploy.json).
+## Deploy via docker
 
-## Prerequisites
+If you would like to deploy via docker-compose please follow the tutorial [deploy-via-docker.md](deploy-via-docker.md) this will deploy DICOM cast as well.
```

Comment:
If you would like to deploy via docker-compose, please follow the quickstart [Deploy Via Docker](deploy-via-docker.md). This will deploy the Medical Imaging Server for DICOM and DICOM Cast.